README.md

```diff
@@ -228,6 +228,11 @@ const options = {
 const result = validate(model, options);
 ```
 
+#### mode
+
+Provides context as to usage mode that data should be validated against.
+For example, to only apply rules that are suitable for data used in a booking flow phase like C1, C2 or B.
+
```

Comment:
@henryaddison 
Is it worth providing an example here like done in the other options' documentation?
Something along the lines of (if at all valid):
```js
const { validate } = require('@openactive/data-model-validator');

const model = {
  type: 'CustomAction'
  // ...
};

const options = {
  mode: ['C2', 'B']
};

const result = validate(model, options);

Reply:
Yes it would